onnxscript/function_libs/torch_lib/_flags.py

```diff
@@ -0,0 +1,11 @@
+"""Experimental flags.
+
+NOTE: These flags are experimental only. Any flag here can be removed at any
+time without notice.
+"""
+
+import os
+
+EXPERIMENTAL_INITIALIZERS_AS_INPUTS: bool = (
```

Comment:
nit: add a comment explaining the behavior?

Reply:
Done. Added as string. PTAL